homeassistant/components/synology_dsm/__init__.py

```diff
@@ -502,7 +505,6 @@ def _async_setup_api_requests(self):
                 "Disable surveillance_station api from being updated for '%s'",
                 self._entry.unique_id,
             )
-            self.dsm.reset(self.surveillance_station)
             self.surveillance_station = None
```

Comment:
Should remove all the condition ?

Reply:
yeah you are right ... this condition is not needed anymore :+1: